<?xml version="1.0"?>
<launch>
    <node name="perception" pkg="perception" type="perception" output="screen">
        <param name="image_topic" value="/carla/camera_1/image_raw"/>
    </node>
</launch> 